### PR TITLE
Changes some gun sounds, adds admin-only bullets type

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About the Pull Request

Changes *some* guns sounds to those from /tg/station.

Added hollow-point SMG bullets that use alternative sound. Currently, those are admin-only.

## Why It's Good For The Game

Some of the guns have rather weird/low quality sounds at the moment and I plan on changing all of them at some point.

## Did you test it?

Tested, shot guns, sounds are okay.

## Changelog

:cl:
soundadd: Fire sounds of some guns have been changed.
tweak: Added hollow-point bullet magazines to the prototype SMG. These are admin-spawn only for now.
/:cl:

### CODE REVIEW

**Comment by Cupax3** [comment]
What does HP stand for?

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -148,13 +148,19 @@
...
 	armor_penetration = 70
 	embed = FALSE
 	distance_falloff = 2
 
+// Higher damage, less AP
+/obj/item/projectile/bullet/flechette/hp
+	fire_sound = 'sound/weapons/gunshot/smg_alt.ogg'
+	damage = 36
+	armor_penetration = 20
+
```
Comment (Cupax3) [inline code/modules/projectiles/projectile/bullets.dm:163]: If these are intended to be hollow point rounds newly added, the PR description is misleading/missing stuff. Please amend.

**Review by Cupax3 — DISMISSED** [review]

**Comment by Shadowtail117** [comment]
> What does HP stand for?

Likely hollow point, a type of bullet that deforms more upon hitting its target, adding more lethality at the expense of penetration capability.

If this is the case, then this PR needs a re-scope.

**Comment by EgorDinamit** [comment]
@Cupax3 I added hollow-point bullets to have a use for alternative SMG sound effect, these are admin only as of now.
Will update PR body.

**Review by Shadowtail117 — DISMISSED** [review]
Even if admin-only, any changes that affect gameplay should be included in the changelog. Something like:
> rscadd: Added hollow-point bullet magazines to the prototype SMG. These are admin-spawn only for now.

would suffice.

**Comment by EgorDinamit** [comment]
@Shadowtail117 okay, done.

**Review by Cupax3 — APPROVED** [review]
LGTM

**Pull request merged**